pkg/virt-launcher/notify-client/client.go

```diff
@@ -177,7 +177,7 @@ func eventCallback(c cli.Connection, domain *api.Domain, libvirtEvent libvirtEve
 			domain.Spec = *spec
 		}
 
-		log.Log.Infof("kubevirt domain status: %v(%v):%v(%v)", domain.Status.Status, status, domain.Status.Reason, reason)
+		log.Log.V(5).Infof("kubevirt domain status: %v(%v):%v(%v)", domain.Status.Status, status, domain.Status.Reason, reason)
```

Comment:
I understand that default for verbosity is 2. Why do you reduce the log verbosity here? I think that in general it would help debugging this in case anything goes wrong here?